openapi/workflow_execution_service.swagger.yaml

```diff
@@ -44,7 +44,7 @@ paths:
   /workflows:
     get:
       summary: |-
-        List the workflows, this endpoint will list the workflows in order of oldest to newest.
+        List submitted workflows.  The ordering of this list is implementation dependent.
```

Comment:
The implementation really has to provide some kind of ordering; maybe implementation-specific. Otherwise, the semantic below of returning to the "first" result has no meaning.